### ISSUE TITLE
wrong estimation of timing

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#967]
how come, with 8 cores and only 4 subjects tested, with average time of 180s, the total time is 2368s?

~~~
348                             Mean      0.25    180.184475  2.250516  0.593410     NaN
349                              STD      0.50     36.709383  1.653938  0.188351     NaN
Passed: 3/4
Total duration: 2368s
Status legend: 0: Passed, 1: Crashed, 99: Failed, 200: File(s) missing
~~~

**Comment by jcohenadad** [comment]
having only two subjects in the folder makes the timing right:
~~~
Testing... (started on: 2016-08-19 12:33:22)
SCT version: dev-228b374dc6092012a1f9f5bf757363af3adf8eca
OS: osx (Darwin-14.5.0-x86_64-i386-64bit)
Hostname: julien-macbook.local
CPU cores: Available: 8, Used by SCT: 8
RAM:
Primary memory available: 16.00 gigabytes
Wired Memory:		1531 MB
Active Memory:		4591 MB
Inactive Memory:	1881 MB
Free Memory:		6796 MB
Results for "sct_label_vertebrae -i t1/t1.nii.gz -s t1/t1_seg_manual.nii.gz -o t1_seg_labeled.nii.gz -c t1 ":
Dataset: /Users/julien/data/test/
                      subject  diff_man  duration [s]  max_dist      rmse  status
0                      23_BAD       NaN           NaN       NaN       NaN     200
1                          27       NaN           NaN       NaN       NaN     200
2           sct_users_wang_01  1.000000    109.503541  4.687500  0.855816      99
3           sct_users_wang_02  0.000000     91.765861  1.657282  0.567104       0
4  xuanwu_20160624_yaou_MS013       NaN           NaN       NaN       NaN     200
5                        Mean  0.500000    100.634701  3.172391  0.711460     NaN
6                         STD  0.707107     12.542434  2.142688  0.204150     NaN
Passed: 1/2
Total duration: 112s
~~~

**Comment by jcohenadad** [comment]
Maybe this is related to when data are hosted in a remote server (accessed via VPN and SMB protocol
)

**Comment by jcohenadad** [comment]
might be related to issue #838

**Comment by jcohenadad** [comment]
ok-- the wrong timing is due to the input file being there, but not the ground truth data!
so the function was in fact ran 35 times with output status 200.
I have now changed test_sct_label_vertebrae to check the existence of the ground truth data BEFORE running the function. This should be done for all test_* functions

**Comment by jcohenadad** [comment]
fixed in b7c5677e108c0e68e7ea3588f926b3b7694cc088
